File: source/v20140727/api/cards/retrieve/_retrieve.rb
Omise.api_key = "skey_test_4xsjvwfnvb2g0l81sjz"

customer = Omise::Customer.retrieve("cust_test_4xsjvylia03ur542vn6")
card = customer.cards.retrieve("card_test_4xsjw0t21xaxnuzi9gs")

# Note that you can reload the card once you have an instance of one.
card.reload

File: source/v20140727/api/cards/retrieve/_retrieve.js
var omise = require('omise-node')({'secretKey': 'skey_test_4xsjvwfnvb2g0l81sjz'});
omise.customers.retrieveCard(
  'cust_test_4xsjvylia03ur542vn6',
  'card_test_4xsjw0t21xaxnuzi9gs',
  function(error, card) {
    /* Response. */
  }
);
